lib/td/command/runner.rb

```diff
@@ -104,16 +105,24 @@ def run(argv=ARGV)
       import_endpoint = e
     }
 
-    op.on('--insecure', "Insecure access: disable SSL (enabled by default)") {|b|
+    op.on('--insecure', "Insecure access: disable SSL certificate verification (enabled by default)") {|b|
```

Comment:
I'm not sure it's a good idea to reuse `--insecure` here. As it just means enable/disable ssl, not certification verification. Changing semantic for this `--insecure` option can cause confusion to both users and maintainers. We can consider deprecate `--insecure` option and remove it in the near future

IMO, it's better to introduce new `--ssl-verify` option that handles both `ssl-verify` and `ssl-ca-file`

Reply:
`--insecure` itself was causing a lot of confusion for our customers in the history.
Some customers try to use insecure option to disable ssl verification and it didn't work as expected as it just change the protocol.
So for me, it is natural to use --insecure for this option. but not that strong desire.
And --insecure is already meaning less as TD enforces https communication in the most case.

> it's better to introduce new --ssl-verify option that handles both ssl-verify and ssl-ca-file

I think this would cause some confusion to merge options into one options as they need to understand one option has 2 meanings.